### PR TITLE
Save transcription process metadata

### PR DESCRIPTION
**Pull request by mushtaqak**
The purpose of this PR is to save save transcription process metadata and show in admin side.

### CODE REVIEW

File: VEDA_OS01/models.py [hunk 1]
```diff
@@ -463,3 +463,23 @@ class Meta:
...
+
+
+class TranscriptionProcessMetadata(TimeStampedModel):
+    """
+    Model to contain third party transcription process metadata.
+    """
+    video = models.ForeignKey(Video)
+    provider = models.CharField('Transcription provider', max_length=50, choices=TranscriptionProviderType.CHOICES)
+    process_id = models.CharField('Process id', max_length=255)
+    lang_code = models.CharField('Language code', max_length=3)
```
Comment (muhammad-ammar) [inline VEDA_OS01/models.py]: `status` field is missing

File: VEDA_OS01/models.py [hunk 2]
```diff
@@ -463,3 +480,29 @@ class Meta:
 
     def __unicode__(self):
         return u'{org} - {provider}'.format(org=self.org, provider=self.provider)
+
+
+class TranscriptProcessMetadata(TimeStampedModel):
```
Comment (muzaffaryousaf) [inline VEDA_OS01/models.py]: can we set plural name for this model as well.

**Comment by mushtaqak** [comment]
@muzaffaryousaf good to merge ?

**Comment by muzaffaryousaf** [comment]
👍

**Pull request merged**